memory/libhugetlbfs.py

```diff
@@ -81,7 +78,7 @@ def setUp(self):
         if os.path.exists('/proc/sys/vm/nr_hugepages'):
             Hugepages_support = process.system_output('cat /proc/meminfo',
                                                       verbose=False,
-                                                      shell=True)
+                                                      shell=True, sudo=True)
```

Comment:
The description for sudo says:
```
    :param sudo: Whether the command requires admin privileges to run,
                 so that sudo will be prepended to the command.
                 The assumption here is that the user running the command
                 has a sudo configuration such that a password won't be
                 prompted. If that's not the case, the command will
                 straight out fail.
```

So, should we capture any errors / exceptions here ?

Reply:
Also, this can be a `genio.read_file()` instead.

Reply:
sure `genio.read_file()` good to use